packages/edit-site/src/components/header-edit-mode/index.js

```diff
@@ -50,22 +51,25 @@ export default function Header( { showIconLabels } ) {
 		isListViewOpen,
 		listViewShortcut,
 		isVisualMode,
-		settings,
 		blockEditorMode,
+		homeUrl,
 	} = useSelect( ( select ) => {
 		const {
 			__experimentalGetPreviewDeviceType,
 			getEditedPostType,
 			isInserterOpened,
 			isListViewOpened,
 			getEditorMode,
-			getSettings,
 		} = select( editSiteStore );
 		const { getShortcutRepresentation } = select( keyboardShortcutsStore );
 		const { __unstableGetEditorMode } = select( blockEditorStore );
 
 		const postType = getEditedPostType();
 
+		const {
+			getUnstableBase, // Site index.
+		} = select( coreStore );
+
```

Comment:
This is a 100% copy-paste from `packages/block-library/src/home-link/edit.js` -- let me know if there is a better way. I am sure there is.